### ISSUE TITLE
loading a save mid-game causes the game to lock up

### ISSUE THREAD

**Issue by BBQGiraffe** [FWGS/xash3d-fwgs#1248]
there are two instances where loading a save mid game causes a lockup, if I try to load a save from the menu it crashes instantly, if I quickload using F7 the save loads but crashes after walking into certain areas, this issue does not happen when loading from the main menu, only while already I'm in a game, when this happens I can't exit with CTRL+C in the command line and must open another terminal to run "kill -9", running on Lubuntu 18.04.06 LTS i686 
[engine.log](https://github.com/FWGS/xash3d-fwgs/files/10786784/engine.log)

**Comment by BBQGiraffe** [comment]
I replicated the bug but this time it entirely crashed
[engine.log](https://github.com/FWGS/xash3d-fwgs/files/10787124/engine.log)

**Comment by a1batross** [comment]
Looks like yet another ref_soft bug.

**Comment by BBQGiraffe** [comment]
yep, tried it again today on my more modern desktop, OpenGL works fine but it crashes on Software
